pkg/skaffold/deploy/kubectl.go

```diff
@@ -109,10 +118,14 @@ func (k *KubectlDeployer) Deploy(ctx context.Context, out io.Writer, builds []bu
 }
 
 func (k *KubectlDeployer) manifestFiles(manifests []string) ([]string, error) {
-	var nonURLManifests []string
+	var nonURLManifests, gcsManifests []string
 	for _, manifest := range manifests {
 		if !util.IsURL(manifest) {
```

Comment:
You can break this in a switch case and do nothing when manifest is an url
e.g.  https://play.golang.org/p/QaInR52gYCG
```suggestion
                switch {
                case util.IsURL(manifest):
                   // do nothing.
                case strings.HasPrefix(manifest, "gs://"):
                        gcsManifests = append(gcsManifests, manifest)
                default:
                         nonURLManifests = append(nonURLManifests, manifest)
              }                  
```